librz/core/cmd_descs/cmd_type.yaml

```diff
@@ -361,3 +361,29 @@ commands:
         cname: type_xrefs_list_all
         summary: List all types used by any function
         args: []
+  - name: tk
```

Comment:
What are *typeclasses*? Please add a "Explanation" section in these commands so the user can get an idea.